presto-hive/src/main/java/com/facebook/presto/hive/PrestoS3FileSystem.java

```diff
@@ -506,9 +507,14 @@ public void seek(long pos)
                 long skip = pos - position;
                 if (skip <= max(in.available(), MAX_SKIP_SIZE.toBytes())) {
                     // already buffered or seek is small enough
-                    if (in.skip(skip) == skip) {
-                        position = pos;
-                        return;
+                    try {
+                        if (in.skip(skip) == skip) {
+                            position = pos;
+                            return;
+                        }
+                    }
+                    catch (IOException e) {
+                        log.info("Got IO exception while trying to seek position, trying to reopen input stream. (%s)", e.toString());
```

Comment:
I don't think we need to log this. The retry code in the `read()` method does not log exceptions. Just rename the exception variable to `ignored` and add a comment:
```java
// will retry by re-opening the stream